app/src/lib/stores/helpers/infer-comparison-branch.ts

```diff
@@ -98,17 +94,27 @@ async function getDefaultBranchOfFork(
     return getMasterBranch(branches)
   }
 
-  const range = revRange(currentBranch.tip.sha, defaultBranch.tip.sha)
-  const aheadBehind = await getAheadBehind(repository, range)
-  if (aheadBehind !== null && aheadBehind.ahead > 0) {
+  // I need to keep track of ahead/behind of two branches and return based on which one has ahead > 0
```

Comment:
:art: can this comment be descriptive about what it's actually looking for below?